Source/Scene/Camera.js

```diff
@@ -801,6 +830,96 @@ define([
         Cartesian3.cross(this.direction, this.up, this.right);
     };
 
+    var scratchSetViewCartesian = new Cartesian3();
+    var scratchSetViewTransform1 = new Matrix4();
+    var scratchSetViewTransform2 = new Matrix4();
+    var scratchSetViewQuaternion = new Quaternion();
+    var scratchSetViewMatrix3 = new Matrix3();
+    var scratchSetViewCartographic = new Cartographic();
+
+    /**
+     * Sets the camera position and orientation with heading, pitch and roll angles.
+     *
+     * The position can be given as either a cartesian or a cartographic. If both are given,
+     * then the cartesian will be used. If neither is given, then the current camera position
+     * will be used.
+     *
+     * @param {Cartesian3} [options.cartesian] The cartesian position of the camera.
```

Comment:
I noticed we didn't change this before we merged.  I agree with @pjcozzi that `position` and `positionCartographic` would probably be better.  Although, since `Cartesian3.fromDegrees` exists, I would just use `position` and be done with it.  Any objections to making this change before we ship?